main.go

```diff
@@ -40,10 +40,10 @@ func (c *Column) ToFieldSchema() *bigquery.FieldSchema {
 	switch c.Type {
 	case "character varying", "text":
 		f.Type = bigquery.StringFieldType
-	case "integer", "bigint":
+	case "integer", "bigint", "smallint":
 		f.Type = bigquery.IntegerFieldType
-	//case "":
-	//	f.Type = bigquery.FloatFieldType
+  case "double precision":
```

Comment:
Want to fix the indentation here? And I'll get this merged. Thank you!

Reply:
Indentation fixed